src/main/java/com/launchcode/recipeproject/controllers/TagsController.java

```diff
@@ -13,7 +13,6 @@
 
 import javax.validation.Valid;
 
-@Controller
```

Comment:
I still need this to have the controller annotation.

Reply:
done